tests/unit/ConditionalHide.spec.js

```diff
@@ -60,43 +60,43 @@ describe('Hide or show controls', () => {
   it('Hide control with evaluation success', () => {
 
     //The fields that not contain a conditional are displayed by default
-    expect(wrapper.vm.showElement.field1).toBe(true);
-    expect(wrapper.vm.showElement.field4).toBe(true);
+    expect(wrapper.contains('#field1')).toBe(true);
```

Comment:
There was also another option of using `expect(wrapper.html()).toContain('<div id="field1"></div>....)` but I thought doing this way would keep the test simipler.